lockable/lockable.py

```diff
@@ -88,7 +88,7 @@ def parse_requirements(requirements_str: (str or dict)) -> dict:
             return json.loads(requirements_str)
         except json.decoder.JSONDecodeError as error:
             # if the first char is not {, try to parse as string requirements
-            if error.colno > 1:
+            if error.pos > 1:
```

Comment:
Comment doesn't match with the implementation:

```
>>> try:
...   json.loads('{')
... except Exception as e:
...   print(e.pos)
... 
1
```

Reply:
perhaps could be just as simple than compare first char - if `{`  then it's json format and otherwise string format..

Reply:
updated here: 330a7575b228283cdd292966e1a18cbacce5cc31